dissect/target/plugins/apps/webservers/iis.py

```diff
@@ -85,6 +87,11 @@ def iter_log_format_path_pairs(self) -> list[tuple[str, str]]:
             for log_file in self.iter_log_paths(log_dir_path):
                 yield (log_format, log_file)
 
+    @plugin.internal
+    def parse_autodetect_format_log(self, path: Path) -> Iterator[BasicRecordDescriptor]:
+        yield from self.parse_iis_format_log(path)
+        yield from self.parse_w3c_format_log(path)
```

Comment:
To my understanding, we now also will get some noise because W3C logs will sort of be parsed when analysed with the IIS format code. So I guess we do need logic here to determine the log file, but I might be wrong.

Main difference is W3C logs are separated with spaces (and all values are quoted if they contain spaces), IIS logs contain commas instead. So maybe this can be used for fast logic?

Reply:
Looking more into the code, it will probably just give a whole lot of debug log lines instead. It will still go over each line in the log, so generating some overhead there. Therefore, I would still suggest to add some logic here.